lib/generators/react_on_rails/install_generator.rb

```diff
@@ -75,6 +76,11 @@ def missing_node?
         GeneratorMessages.add_error(error)
         true
       end
+
+      def add_bin_scripts
+        directory "#{__dir__}/bin", "bin"
+        chmod "bin", 0o755 & ~File.umask, verbose: false
```

Comment:
Why is this needed?

Reply:
If you mean the chmod line, I got the idea from the code in shakapacker [binstubs](https://github.dev/shakacode/shakapacker/blob/30044b5a610ce9d0edb5eeb3de04903851c6e3ad/lib/install/binstubs.rb#L4) file. I thought about it and the only guess I was about issues might occur in Windows machines by loosing execution flag. But that also doesn't make much sense since `rails generate react_on_rails:install` runs once. And I believe it is a bad idea to run a command that can affect other irrelevant files.

I tried to go with a simpler solution of adding `bin/dev` to the root of the gem (as I read in the rubygem documentation) but it didn't work. So I took a look at shakapacker code and tried to get the working solution from there.

Reply:
I removed `chmod` since I couldn't find any good reason to use it in this case. The file in the `bin` directory is already executable.